src/test/resources/application-test.yml

```
@@ -10,4 +10,4 @@ spring:
         show_sql: true
         format_sql: true
     hibernate:
-      ddl-auto: none
\ No newline at end of file
+      ddl-auto: create
```

```suggestion
      ddl-auto: update
```

혹시 만에하나 실수로 파일 옮기다가 실수로 운영환경으로 올릴수도 있기에 그나마 안전한 update를 추천합니다.